scripts/performance/cpu-performance/synthetic-trigger.js

```diff
@@ -0,0 +1,53 @@
+const { fetchHandlingError } = require('../../lib/execution-utils')
+const { getOrg2ApiKey, getOrg2AppKey } = require('../../lib/secrets')
+const { timeout } = require('../../lib/execution-utils')
+const API_KEY = getOrg2ApiKey()
+const APP_KEY = getOrg2AppKey()
+const TIMEOUT_IN_MS = 10000
+const TEST_PUBLIC_ID = 'vcg-7rk-5av'
+
+async function syntheticTrigger(prNumber, commitId) {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏‏for readability sake, it could be nice to use consistent form / naming: 
- We mostly use `doSomething` form instead of `somethingDo`, so `triggerSynthetic` could be more aligned with other namings
- Looking at the other function `getSyntheticTestResult`, it could be nice to use the same naming for the same thing and so use either `synthetic` or `syntheticTest` in both function names (and in file name)
- looking at product and API name, it is actually synthetics not synthetic